### PR TITLE
ES: Translation for security

### PR DESCRIPTION
**Pull request by UlisesGascon**
This is my first PR to do a full page translation. I am not sure 100% for some terms translation to Spanish but I tried to be very literal and I keep some english terms that are hard to translate to proper Spanish.

Please review it @nodejs/nodejs-es 🙏

### Main changes
- Updated `locale/es/docs/guides/index.md` to match `locale/en/docs/guides/index.md` content
- Added Spanish translation for Security Guidelines

### Context

- Refs: https://github.com/nodejs/nodejs.org/pull/4896.
- [Slack discussion](https://openjs-foundation.slack.com/archives/C03Q9MS3KFB/p1668269451339869)

### CODE REVIEW

**Comment by ghost** [comment]
Ping @nodejs/nodejs-es , plz have a review.

**Review by AugustinMauroy — APPROVED** [review]
✨LGTM✨

File: locale/es/docs/guides/security/index.md [hunk 1]
```diff
@@ -0,0 +1,334 @@
...
+
+# Buenas prácticas de seguridad en Node.js
+
+## Propósito
+
+Este documento tiene la intención de extender el [modelo de amenazas][] actual y proporcionar directrices para asegurar una aplicación Node.js.
+
+## Contenido del documento
+* Mejores prácticas: Mostramos una forma simplificada y resumida de ver las mejores prácticas. Podemos usar [este issue][issue sobre la guía de seguridad] o [esta guía][nodejs guideline] como punto de partida. Es importante señalar que este documento es específico a Node.js, si está buscando algo amplio, considere [OSSF Best Practices][].
+* Ataques explicados: ilustrar y documentar con lenguaje sencillo con algunos ejemplos código
```
Comment (vrunoa) [inline locale/es/docs/guides/security/index.md]: **suggestion:** `Ataques explicados: ilustrar y documentar con lenguaje sencillo con algunos ejemplos código`  --> `Ataques explicados: ilustrar y documentar en lenguaje sencillo con algunos ejemplos código`.

File: locale/es/docs/guides/security/index.md [hunk 2]
```diff
@@ -0,0 +1,334 @@
...
+  (cuando sea posible) los ataques que estamos mencionando en el modelo de amenazas.
```
Comment (vrunoa) [inline locale/es/docs/guides/security/index.md]: **suggestion:** `los ataques que estamos mencionando en el modelo de amenazas.` --> `los ataques mencionandos en el modelo de amenazas.`

File: locale/es/docs/guides/security/index.md [hunk 3]
```diff
@@ -0,0 +1,334 @@
...
+
+Node.js tiene un [policy mechanism][] que es experimental[¹][experimental-features] para declarar el recurso cargado como no fiable o fiable. Sin embargo, esta política no está habilitada por defecto.
+
+Asegúrate de fijar las versiones de las dependencias y de ejecutar comprobaciones automáticas de vulnerabilidades utilizando flujos de trabajo comunes o através de scripts de Npm.
+
+Antes de instalar un paquete asegúrese de que éste esta mantenido y que incluye todo el contenido que esperaba. Tenga cuidado, ya que el código fuente en Github no siempre es el mismo que el publicado en Npm, valídelo en el _node\_modules_.
+
+#### Ataques a la cadena de suministro
+
+Un ataque a la cadena de suministro en una aplicación Node.js, ocurre cuando una de sus dependencias (ya sea directa o transitiva) está comprometida. Esto puede ocurrir debido a que la aplicación es demasiado laxa en la especificación de las dependencias (permitiendo actualizaciones no deseadas) y/o errores comunes en la especificación (vulnerable a [typosquatting][]).
```
Comment (vrunoa) [inline locale/es/docs/guides/security/index.md]: **suggestion:** `la aplicación es demasiado laxa` --> `la aplicación no es estricta`

**Review by vrunoa — APPROVED** [review]
Small suggestions

**Comment by ovflowd** [comment]
@AugustinMauroy I know you have the best intentions with your LGTM, but if you're not part of the Website WG or not a Spanish translator, avoid stamping LGTM's as it gives a wrong impression that this PR got approved 🙈

File: locale/es/docs/guides/security/index.md (hunk 1 above)
Comment (UlisesGascon) [reply locale/es/docs/guides/security/index.md]: Fix in [c4619df](https://github.com/nodejs/nodejs.org/pull/4932/commits/c4619df7c402460da9dff8c71ce9e3205de81181)

File: locale/es/docs/guides/security/index.md (hunk 2 above)
Comment (UlisesGascon) [reply locale/es/docs/guides/security/index.md]: Fixed in [c4619df](https://github.com/nodejs/nodejs.org/pull/4932/commits/c4619df7c402460da9dff8c71ce9e3205de81181)

File: locale/es/docs/guides/security/index.md (hunk 3 above)
Comment (UlisesGascon) [reply locale/es/docs/guides/security/index.md]: Fixed in [c4619df](https://github.com/nodejs/nodejs.org/pull/4932/commits/c4619df7c402460da9dff8c71ce9e3205de81181)

**Comment by UlisesGascon** [comment]
Thanks for the suggestions @vrunoa! 🙌

**Review by eafelix — APPROVED** [review]

**Comment by UlisesGascon** [comment]
Can we merge it? :)

**Review by vrunoa — APPROVED** [review]

**Comment by vrunoa** [comment]
@UlisesGascon I think someone from @nodejs/website still needs to approve it ? or maybe @nodejs/translators ?

**Comment by UlisesGascon** [comment]
mmm... not sure, first time I do a translation 🤔

**Review by ovflowd — APPROVED** [review]

**Pull request merged**